src/EA.Iws.Core/Shared/ShipmentAuditRecord.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace EA.Iws.Core.Shared
+{
+    using System;
+
+    public class ShipmentAuditRecord
+    {
+        public ShipmentAuditRecord()
+        {
+        }
+
+        public ShipmentAuditRecord(int shipmentNumber, string auditType, string userName,  DateTimeOffset dateAdded)
+        {
+            this.UserName = userName;
+            this.ShipmentNumber = shipmentNumber;
+            this.AuditType = auditType;
+            this.DateAdded = dateAdded;
+        }
+
+        public string UserName { get; set; }
+
+        public int ShipmentNumber { get; set; }
+
+        public string AuditType { get; set; }
```

Comment:
Can we change this into the `enum` type rather than `string`. It'll give us flexibility to change the name to be displayed on the UI - the movement status is a perfect example!

Reply:
Good catch...